rstudio/rstudio
# fix overlocalization in import dataset

### Intent

Addresses [Unable to change variable type in the "import dataset" wizard when localization is french #14224](https://github.com/rstudio/rstudio/issues/14224)

### Approach

Revert back to the hardcoded strings that are used to match against data types; these shouldn't have been localized in [this PR](https://github.com/rstudio/rstudio/pull/10229). The strings shown in the dropdown are still localized in `DataImportColumnTypesMenu.ui.xml`.

Note: the non-localization of the types shown in the column headers, e.g. (character) and (double) is an existing issue; I opened https://github.com/rstudio/rstudio/issues/14233 to track it.

<img width="355" alt="import-dataset-types" src="https://github.com/rstudio/rstudio/assets/10569626/2ef5d4e3-9267-41a6-8e64-623afee437a5">

### Automated Tests

None

### QA Notes

Follow steps in the issue and confirm you can now open the dropdown, and it functions as expected, in both English and French.

### Documentation

None

### Checklist
